Documentation/getting-started-guide.md

```diff
@@ -9,19 +9,32 @@ If you're not on Linux, you should do all of this inside [the rkt Vagrant](https
 package main
 
 import (
-	"log"
-	"net/http"
+        "github.com/coreos/go-systemd/daemon"
+        "log"
+        "net"
+        "net/http"
```

Comment:
nit: sort imports, such that stdlib imports come first, and external imports next, like here https://github.com/coreos/rkt/blob/master/rkt/api_service.go#L18 (just took some sample out of the blue)